src/App.js

```diff
@@ -16,9 +17,14 @@ function App() {
     localStorage.setItem("user is logged", loggedStatus)
   }, [loggedStatus])
 
+  //I have placed the breadcrumb component inside the App.js in order to be visible from everywhere and display changes.
```

Comment:
We want it inside the logged in pages only. Breadcrumb is not displayed in the login or the register pages. 
You can place it in `src/Pages/index.js`